src/components/Modals/FiatRamps/views/Overview.tsx

```diff
@@ -133,7 +133,8 @@ export const Overview: React.FC<OverviewProps> = ({
     if (!accountMetadata) return
     const { accountType, bip44Params } = accountMetadata
     const showOnDevice = true
-    const payload = { accountType, bip44Params, wallet, showOnDevice }
+    const { accountNumber } = bip44Params
```

Comment:
Use `selectAccountNumberByAccountId` instead of selecting bip44Params and destructuring it?

Reply:
this would be ideal, but i was attempting to keep the diff as minimal as possible to minimize risk